rotkehlchen/assets/asset.py

```diff
@@ -320,6 +333,9 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090'): 'CPC',
     'ONE-2': 'ONE',
     strethaddress_to_identifier('0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577'): 'YFDAI',
+    'BTMX': 'ASD',
```

Comment:
What's this? I don't see any BTMX asset in the DB

Reply:
Is in the new one 
![image](https://user-images.githubusercontent.com/5068010/125605712-bb61b68e-c943-4026-936b-8aee519314f5.png)

Reply:
Well no its' not.

What you wrote in there is searching for an asset with an identifier called "BTMX". The identifier is `_ceth_0xcc...` as per your screenshot.